RecoParticleFlow/Configuration/python/RecoParticleFlow_EventContent_cff.py

```diff
@@ -5,101 +5,10 @@
-    'keep recoSuperClusters_particleFlowEGamma_*_*',
-    'keep recoConversions_particleFlowEGamma_*_*',
-    'keep recoPFCandidates_particleFlow_*_*',
-    'keep recoPFCandidates_particleFlowTmp_*_*',
-    'drop recoPFCandidates_particleFlowTmp__*',
-    #'keep recoPFDisplacedVertexs_*_*_*',
-    'keep recoPFDisplacedVertexs_particleFlowDisplacedVertex_*_*',
-    'keep *_pfElectronTranslator_*_*',
-    'keep *_pfPhotonTranslator_*_*',
-    'keep *_particleFlow_electrons_*',
-    'keep *_particleFlow_photons_*',
-    'keep *_trackerDrivenElectronSeeds_preid_*',
-    'keep *_particleFlowPtrs_*_*',
-    'keep *_particleFlowTmpPtrs_*_*',
-    'keep *_chargedHadronPFTrackIsolation_*_*'
-        )
-    )
-# RECO content
-RecoParticleFlowRECO = cms.PSet(
-    outputCommands = cms.untracked.vstring(
-    'drop CaloTowersSorted_towerMakerPF_*_*',
-    #'keep recoPFRecHits_*_Cleaned_*',
-    'keep recoPFRecHits_particleFlowClusterECAL_Cleaned_*',
-    'keep recoPFRecHits_particleFlowClusterHCAL_Cleaned_*',
-    'keep recoPFRecHits_particleFlowClusterHO_Cleaned_*',
-    'keep recoPFRecHits_particleFlowClusterHF_Cleaned_*',
-    'keep recoPFRecHits_particleFlowClusterPS_Cleaned_*',
-    'keep recoPFRecHits_particleFlowRecHitECAL_Cleaned_*',
-    'keep recoPFRecHits_particleFlowRecHitHBHE_Cleaned_*',
-    'keep recoPFRecHits_particleFlowRecHitHF_Cleaned_*',
-    'keep recoPFRecHits_particleFlowRecHitHO_Cleaned_*',
-    'keep recoPFRecHits_particleFlowRecHitPS_Cleaned_*',
-    #'keep recoPFClusters_*_*_*',
-    'keep recoPFClusters_particleFlowClusterECAL_*_*',
-    'keep recoPFClusters_particleFlowClusterHCAL_*_*',
-    'keep recoPFClusters_particleFlowClusterHO_*_*',
-    'keep recoPFClusters_particleFlowClusterHF_*_*',
-    'keep recoPFClusters_particleFlowClusterPS_*_*',
-    #'keep recoPFBlocks_*_*_*',
-    'keep recoPFBlocks_particleFlowBlock_*_*',
-    #'keep recoPFCandidates_*_*_*',
-    'keep recoPFCandidates_particleFlowEGamma_*_*',
-    'keep recoCaloClusters_particleFlowEGamma_*_*',
-    'keep recoSuperClusters_particleFlowEGamma_*_*',
-    'keep recoConversions_particleFlowEGamma_*_*',
-    'keep recoPFCandidates_particleFlow_*_*',
-    'keep recoPFCandidates_particleFlowTmp_electrons_*',
-    'keep recoPFCandidates_particleFlowTmp_*_*',
-    'drop recoPFCandidates_particleFlowTmp__*',
-    #'keep recoPFDisplacedVertexs_*_*_*',
-    'keep recoPFDisplacedVertexs_particleFlowDisplacedVertex_*_*',
-    'keep *_pfElectronTranslator_*_*',
-    'keep *_pfPhotonTranslator_*_*',
-    'keep *_particleFlow_electrons_*',
-    'keep *_particleFlow_photons_*',
-    'keep *_particleFlow_muons_*',
-    'keep *_trackerDrivenElectronSeeds_preid_*',
-    'keep *_particleFlowPtrs_*_*',
-    'keep *_particleFlowTmpPtrs_*_*',
-    'keep *_chargedHadronPFTrackIsolation_*_*'
-        )
-)
-
 # AOD content
 RecoParticleFlowAOD = cms.PSet(
     outputCommands = cms.untracked.vstring(
     'drop CaloTowersSorted_towerMakerPF_*_*',
```

Comment:
No, it isn't really used so it can be deleted. There is no difference after remove this drop line.
(The towerMakerPF doesn't appear to event contents.)